README.md

```diff
@@ -203,6 +267,20 @@ the coding standard we decided to adhere to in this project.
     yarn coding-standards-check
     ```
 
+## Tests
+
+The following command let you run the bundle tests
+
+```shell
+./vendor/bin/phpunit --testdox tests
+```
+
+The tests are based on one specific
+[cache adapter](https://symfony.com/doc/current/components/cache.html#available-cache-adapters),
+namely the
+[Filesystem Cache Adapter](https://symfony.com/doc/current/components/cache/adapters/filesystem_adapter.html),
+to make sure the bundle is behaving correctly.
```

Comment:
If using https://symfony.com/doc/current/components/cache/adapters/array_cache_adapter.html instead of an filesystem adapter, you could omit this part.